diagnosis_api/tests/test_disease_details_routes.py

```diff
@@ -5,36 +5,44 @@
 
 
 class TestApiRoutes(unittest.TestCase):
-
     def setUp(self):
         self.app_client = app.test_client()
-    
+
     def test_index(self):
 
         index = self.app_client.get("/index", content_type="application/json")
         self.assertEqual(index.status_code, 200)
         response = json.loads(index.data)
-        self.assertEqual(response['message'], 'Welcome to disease diagnosis API with pandas')
+        self.assertEqual(
+            response["message"], "Welcome to disease diagnosis API with pandas"
+        )
 
     def test_index1(self):
-     
+
         index = self.app_client.get("/", content_type="application/json")
         self.assertEqual(index.status_code, 200)
         response = json.loads(index.data)
-        self.assertEqual(response['message'], 'Welcome to disease diagnosis API with pandas')
+        self.assertEqual(
+            response["message"], "Welcome to disease diagnosis API with pandas"
+        )
 
     def test_api_gets_disease_details(self):
-        disease_details = self.app_client.get("api/v1/get_disease_details?file_name =rabbit_diseases.xls&disease_name=snuffles", content_type="application/json")
+        disease_details = self.app_client.get(
+            "/api/v1/diseases/details?file_name =rabbit_diseases.xls&disease_name=snuffles",
+            content_type="application/json",
+        )
         self.assertEqual(disease_details.status_code, 200)
         response = json.loads(disease_details.data)
-        self.assertIsInstance(response,dict)
+        self.assertIsInstance(response, dict)
 
     def test_api_searches_disease_by_symptoms(self):
-        disease = self.app_client.get("api/v1/get_disease_by_sypmtoms?file_name =rabbit_diseases.xls&search_params=['nasal discharge']", content_type="application/json")
+        disease = self.app_client.get(
+            "/api/v1/diseases/search?file_name =rabbit_diseases.xls&search_params=['nasal discharge']",
```

Comment:
I'm not sure why you have square brackets in the params . Usually params are formed as key value. Something like
`some/url?q=louis&sex=male&confirmed=true`. Again, I'm not completely sure about this and so I will do more research about it and see if I find something else.

Reply:
The reason why i am passing the query parameters as a list is because for any disease, the symptoms are usually many, for instance, if you go to a doctor for consultant, they normally ask you how you are feeling, then you will tell the doctor that you have say (headache, dizziness, fever, stomachache) . Depending on his experience, he will return all disease that much that contain a combination of all or some of the symptoms as the most likely illness. That is why i thought it would be appropriate to pass that as a list.

Reply:
i was also some puzzled by how to pass the multiple instances of a single parameter for example 
some/url?symptoms1=somesymptom&symptoms2=somesymptom&symptoms3=somesymptom
which are all talking about symptoms for a single disease.
It did not much sense to me. you could try to shed some light if it does.